src/ck-core/init.C

```diff
@@ -1558,9 +1558,9 @@ void _initCharm(int unused_argc, char **argv)
 	}
 
 	/* The following will happen on every virtual processor in BigEmulator, not just on once per real processor */
-	if (CkMyRank() == 0) {
+	//	if (CkMyRank() == 0) {
```

Comment:
What's the use of this? We should avoid adding commented-out code that is not debug print statements, and ideally all debug print statements would be controlled by macros rather than commented out

Reply:
Good catch, i had commented it out while determining the necessary extent of Rank 0 restrictions.  I'll add a fixup to remove that line.

Reply:
added in fixup